Description: Remove q3lcc
 q3lcc is non-free, and has been removed from the Debianized upstream tarball.
 Upstream themselves will remove it from the next release. So adopt
 CMakeLists.txt to not build with it.
Author: Lee Garrett <debian@rocketjump.eu>
Forwarded: https://github.com/QW-Group/ktx/pull/257
Last-Update: 2023-01-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -153,14 +153,12 @@
 # Build QVM tools.
 # EXCLUDE_FROM_ALL tells to not build tools if they does not required, for example if you build native library only.
 add_subdirectory(tools/q3asm EXCLUDE_FROM_ALL)
-add_subdirectory(tools/q3lcc EXCLUDE_FROM_ALL)
 
 # Set up vars for QVM library.
 set(VM_DIR "vm")
 add_custom_target(vmdir
 	COMMAND ${CMAKE_COMMAND} -E make_directory ${VM_DIR}
 	)
-set(QVM_COMPILER "q3lcc")
 set(QVM_LINKER "q3asm")
 set(QVM_C_FLAGS -DQ3_VM -S -Wf-target=bytecode -Wf-g)
 if(BOT_SUPPORT)
@@ -180,7 +178,7 @@
 	file(APPEND ${QVM_LIST_FILE} "${VM_DIR}/${file_asm}\n") # Continue generate QVM_LIST_FILE.
 	# Add command to compile c to asm.
 	add_custom_target(${file_asm}
-		COMMAND ${QVM_COMPILER} ${QVM_C_FLAGS} ${QVM_INCFLAGS} "${CMAKE_CURRENT_SOURCE_DIR}/${file}" -o "${VM_DIR}/${file_asm}"
+		COMMAND ${CC} ${QVM_C_FLAGS} ${QVM_INCFLAGS} "${CMAKE_CURRENT_SOURCE_DIR}/${file}" -o "${VM_DIR}/${file_asm}"
 		DEPENDS vmdir
 		SOURCES ${file}
 	)
